spec/models/user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
For model specs, this gem would be beneficial: https://github.com/thoughtbot/shoulda-matchers